docs/defaultConstructor.md

```diff
@@ -5,29 +5,29 @@ Provides a constructor function to be used to provide a default value for a prop
 
 @signature `Default`
 
-A constructor function can be provided that is called to create a default value used for this property.
-This constructor will be invoked with `new` for each created instance. The default
-value is created on demand when the property is read for the first time.
-
-Specify `Default` like:
-
-```js
-{
-	prop: {
-		Default: Array
-	},
-	person: {
-		Default: Person
-	}
-}
-```
+  A constructor function can be provided that is called to create a default value used for this property.
+  This constructor will be invoked with `new` for each created instance. The default
+  value is created on demand when the property is read for the first time.
+
+  Specify `Default` like:
+
+  ```js
+  {
+    prop: {
+      Default: Array
+    },
+    person: {
+      Default: Person
+    }
+  }
+  ```
 
 @body
 
 ## Use
 
```

Comment:
Add some text here about what the following example shows.

Reply:
Actually this example doesn't even use `Default`.  Please make one that does.

Reply:
What would that text look like?
> In the example below we can see that `value` properties from the `Address` constructor are being set as the default information in `Direction` instances